crates/starlight/src/vm/value.rs

```diff
@@ -680,6 +680,8 @@ impl JsValue {
             }
             println!("{:?}", (object.get_dyn()).type_name());
             todo!()
+        } else if self.is_native_value() {
+            Ok(self.get_native_u32().to_string())
```

Comment:
This one should not be printed. It is internal value type instead panic should be triggered

Reply:
ok~